core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -81,6 +83,7 @@ public class FeaturesConfig
      * default value is overwritten for fault tolerant execution in {@link #applyFaultTolerantExecutionDefaults()}}
      */
     private boolean exchangeCompressionEnabled;
```

Comment:
We should deprecate the boolean flag and instead use NONE enum value to disable compression.